model-optimizer/extensions/middle/L2NormFusing.py

```diff
@@ -82,16 +82,31 @@ def replace_pattern(self, graph: Graph, match: dict):
             log.debug('The value of the "maximum_y_data" is not defined or is not constant')
             return
```

Comment:
I think that it is better to call this transformation as `FusionToNormalizeL2`

Reply:
btw, do we really need this transformation in MO? Why don't we make it by nGraph?

Reply:
Decided to not apply the comment.